src/main/java/seedu/duke/Duke.java

```diff
@@ -4,11 +4,48 @@
 import seedu.duke.logic.parser.Parser;
 import seedu.duke.ui.Ui;
 
+import java.util.ArrayList;
+
 public class Duke {
 
     private static Parser parser;
+    private static ArrayList<Bus> busesInZoneA = new ArrayList<>();
+    private static final String[] ZONE_A = {"AA1", "AA2"};
+
+    private static final String[] AA1 = {
+        "PGP",
+        "Kent Ridge MRT Station",
+        "Opp University Health Center",
+        "Yusof Ishak House",
+        "Central Library",
+        "Kent Ridge",
+        "Museum",
+        "University Town",
+        "University Health Centre",
+        "Opp Kent RIdge MRT station",
+        "PGPR"
+    };
+
+    private static final String[] AA2 = {
+        "PGP",
+        "Kent Ridge MRT Station",
+        "Opp University Health Centre",
+        "University Town",
+        "Raffles Hall",
+        "Kent Vale",
+        "EA",
+        "NUS IT",
+        "University Health Centre",
+        "Opp Kent Ridge MRT Station",
+        "PGPR"
+    };
 
     public Duke() {
+        //open file
+        //busesInZoneA = fileManager.loadData();
+        busesInZoneA.add(new Bus("AA1", AA1));
+        busesInZoneA.add(new Bus("AA2", AA2));
```

Comment:
Could refactor into a method adding all bus routes to the arraylist of bus object. For loop to run through and add the bus routes into the array list. This is to prepare for larger number of bus routes as we include other zones in our future versions. Keep in view for future commits.
```suggestion
        busesInZoneA.add(new Bus(Zone_A[0], AA1));
        busesInZoneA.add(new Bus(Zone_A[1], AA2));
```